### PR TITLE
fix: increase text contrast on level table

### PR DESCRIPTION
**Pull request by Ali7040**
Issue #862

### CODE REVIEW

**Comment by actuallyyun** [comment]
Hey @Ali7040 great to see that you have fixed the errors in the test! 
Could you add a screenshot of the changes you made in the PR description? In your case, it would be the UI you changed. 
Thank you!

**Comment by Ali7040** [comment]
image

in Dtable test file

**Comment by Ali7040** [comment]
image

In Dtable file

**Comment by vnugent** [comment]
Sorry I wasn't clear.  Can you show a screenshot of the website?  If you scroll up a bit you can find the Preview build link.

---
Screenshot 2023-07-02 at 5 37 03 AM

**Comment by Ali7040** [comment]
image

 local host screenshot

File: src/components/ui/DTable.tsx [hunk 1]
```diff
@@ -85,10 +85,10 @@ const Row = ({ rowHeader, beginner, intermediate, advanced, expert, total, myRan
   return (
     <tr className='text-tertiary text-xs my-2'>
       <th scope='row' className='py-0.5 pr-2 text-right'>{rowHeader}</th>
-      <td className={`${highlighted ? 'dtable-highlight' : ''} ${highlighted && isInMyRange(0, myRange) ? 'dtable-my-range' : ''}`}>{beginner}</td>
-      <td className={`${highlighted ? 'dtable-highlight' : ''} ${highlighted && isInMyRange(1, myRange) ? 'dtable-my-range' : ''}`}>{intermediate}</td>
-      <td className={`${highlighted ? 'dtable-highlight' : ''} ${highlighted && isInMyRange(2, myRange) ? 'dtable-my-range' : ''}`}>{advanced}</td>
-      <td className={`${highlighted ? 'dtable-highlight' : ''} ${highlighted && isInMyRange(3, myRange) ? 'dtable-my-range' : ''}`}>{expert}</td>
```
Comment (vnugent) [inline src/components/ui/DTable.tsx]: In this case I think it's better to update the `dtable` and `dtable-highlight` classes in  [src/styles/defaults.css](https://github.com/OpenBeta/open-tacos/blob/develop/src/styles/defaults.css#L411-L417).  

This way you preserve the original intent of the code and the DTable unit test.

Something like this

```css
table td {
  @apply my-2 text-base-content !important;
}

table td.dtable-highlight {
  @apply text-sm  bg-gray-200;
}
```

For reference you can find DaisyUI semantic color classes here:  https://daisyui.com/docs/colors/

Some of the components were developed before we adopted DaisyUI so you see TailwindUI color classes.  We should avoid that if possible.

**Review by vnugent — CHANGES_REQUESTED** [review]

File: src/components/ui/DTable.tsx (hunk 1 above)
Comment (Ali7040) [reply src/components/ui/DTable.tsx]: I did, but when I tried to save default.css the file, it gave me an error. That's why I approached it this way.

File: src/styles/defaults.css [hunk 2]
```diff
@@ -409,12 +409,19 @@ input:checked + label {
 
 /* DTable (grade distribution table */
 table td {
+    @apply my-2 text-base-content !important;
+  }
+  
+  table td.dtable-highlight {
+    @apply text-sm  bg-gray-200;
+  }
+/* table td {
```
Comment (vnugent) [inline src/styles/defaults.css]: can you please remove the commented out css?  we're good to merge after that!

**Review by vnugent — CHANGES_REQUESTED** [review]

File: src/styles/defaults.css (hunk 2 above)
Comment (Ali7040) [reply src/styles/defaults.css]: Done!

**Pull request merged**